api-tools/webhooks/nodejs/webhookserver.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+const bodyParser = require('body-parser')
+
+//const fs = require('node:fs');
+
+var jsonParser = bodyParser.json()
+
+app.listen(port, () => {
+  console.log(`Server listening on port ${port}`);
+});
```

Comment:
This should be moved to the bottom. First define the routes and then call `app.listen`